deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1405,10 +1405,13 @@ def get_fim_mode_param(mode, key='FIM_MODE'):
     """
     metadata = {key.lower(): mode}
     if mode == 'scheduled':
-        return {key: ''}, metadata
-    elif mode == 'realtime' and sys.platform not in _os_excluded_from_rt_wd:
+        if mode in global_parameters.fim_mode:
```

Comment:
This condition is used in all three if-elif checks, please extract it outside of the block to improve readability.

It would look something like this:
```
    if mode not in global_parameters.fim_mode:
        return None, None

    if mode == 'scheduled':
        return {key: ''}, metadata
    elif mode == 'realtime' and sys.platform not in _os_excluded_from_rt_wd:
        return {key: {'realtime': 'yes'}}, metadata
    elif mode == 'whodata' and sys.platform not in _os_excluded_from_rt_wd:
        return {key: {'whodata': 'yes'}}, metadata
    else:
        return None, None
```